commit/merkleroot/processor.go

```diff
@@ -15,6 +15,10 @@ import (
 	"github.com/smartcontractkit/chainlink-ccip/shared"
 )
 
+// DefaultSeqNumsBatchLimit is the default number of max new messages to scan, we use this value when
+// the config is not set for a specific chain.
+const DefaultSeqNumsBatchLimit = 256
```

Comment:
[MaxNumberTreeLeaves](https://github.com/smartcontractkit/chainlink-common/blob/e78a0de3f6847845b18da1797e5dccf9521abed2/pkg/merklemulti/merkle_multi.go#L35)

```suggestion
const DefaultSeqNumsBatchLimit = merklemulti.MaxNumberTreeLeaves
```

Reply:
Isn't this the default unless there's an override? Or you're saying there should be no default and the system needs to crash if the config is missing?

Reply:
Ideally the latter (IMO defaults are often more trouble than they're worth in this scenario, because people would just use the default even though it might be wrong) but to start with we could fall back to the default and enforce that its provided later if we decide to do so

Reply:
Added EVM prefix and moved to pluginconfig package